From 50fec34283ce34caeb7b554a6c241218234746b3 Mon Sep 17 00:00:00 2001
From: Johannes Pointner <johannes.pointner@br-automation.com>
Date: Fri, 3 May 2019 09:12:38 +0200
Subject: [PATCH] chromium: Do not try to set the guessed values for

 march/mtune/float-abi OE config machinary has computed these values already
 and fed it via compiler cmdline to chromium build

I think upstream should check for the values on compiler cmdline
before overriding these

Upstream-Status: Inappropriate [OE-Specific]

Signed-off-by: Khem Raj <raj.khem@gmail.com>
Signed-off-by: Johannes Pointner <johannes.pointner@br-automation.com>

%% original patch: 0005-chromium-Do-not-try-to-set-the-guessed-values-for-ma.patch
---
 chromium/build/config/compiler/BUILD.gn | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/chromium/build/config/compiler/BUILD.gn b/chromium/build/config/compiler/BUILD.gn
index c007eab8ec9..feb4340f522 100644
--- a/chromium/build/config/compiler/BUILD.gn
+++ b/chromium/build/config/compiler/BUILD.gn
@@ -783,15 +783,6 @@ config("compiler_cpu_abi") {
     } else if (current_cpu == "arm") {
       if (is_clang && !is_android && !is_nacl) {
       }
-      if (!is_nacl) {
-        cflags += [
-          "-march=$arm_arch",
-          "-mfloat-abi=$arm_float_abi",
-        ]
-      }
-      if (arm_tune != "") {
-        cflags += [ "-mtune=$arm_tune" ]
-      }
     } else if (current_cpu == "arm64") {
       if (is_clang && !is_android && !is_nacl && !is_fuchsia) {
       }
